backend/validators/management/commands/rebuild_daily_snapshots.py

```
@@ -0,0 +1,93 @@
+
+
+class Command(BaseCommand):
+    help = 'Rebuild daily validator status rollups from the raw observation log'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--days', type=int, default=None,
+            help='Only rebuild the last N days of rollups (default: all observations)'
+        )
+
+    def handle(self, *args, **options):
```

Ruff (0.15.20)

[warning] Missing type annotation for `*args`

(ANN002)